iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/telemetry/SendMessagesTests.java

```diff
@@ -58,12 +56,6 @@ public void sendSecurityMessages() throws Exception
     @Test
     public void sendMessages() throws Exception
     {
-        if (testInstance.protocol == MQTT_WS && (testInstance.authenticationType == SELF_SIGNED || testInstance.authenticationType == CERTIFICATE_AUTHORITY))
```

Comment:
removed, why?

Reply:
MQTT_WS does support x509 auth now. It's orthogonal to this PR, but worth enabling